resources/testharness.js

```diff
@@ -543,8 +543,8 @@ policies and contribution forms [3].
                 if (trimmed) {
                     // add a suffix if we already have this string
                     if (seen_func_name[trimmed]) {
-                        trimmed = trimmed + " " + seen_func_name[trimmed];
-                        seen_func_name[trimmed]++;
+                        let current_count = seen_func_name[trimmed]++;
```

Comment:
Can we avoid using `++` here. Depending on details of post increment vs pre increment seems to be quite bug prone. Something like 
```
if (trimmed) {
  let name = trimmed
  if (seen_func_name[trimmed]) {
    // This base name exists, so add a suffix
    name += " " + seen_func_name[trimmed];
  } else {
    seen_func_name[trimmed] = 0;
  }
  seen_func_name[trimmed] += 1;
  return name;
}

Reply:
Good point; I agree. Fixed.